lib/addons/secrets-store/csi-driver-provider-aws-secrets.ts

```diff
@@ -95,13 +100,15 @@ export enum KubernetesSecretType {
 interface ParameterObject {
     objectName: string;
     objectType: string;
+    objectAlias?: string;
     jmesPath?: JmesPathObject[];
 }
 
-function createParameterObject(csiSecret: CsiSecretProps, secretName: string, secretType: AwsSecretType) {
+function createParameterObject(csiSecret: CsiSecretProps, secretName: string, secretType: AwsSecretType, secretAlias?: string) {
     const result: ParameterObject = {
         objectName: secretName,
         objectType: secretType,
+        objectAlias: secretAlias,
```

Comment:
have you tested it on a case when there is no objectAlias, e.g. the existing functionality?
My concern is that the above structure will generate an empty objectAlias and may not be treated the same as lack of attribute. 

the safest is to add objectAlias only when secretAlias is defined. Unless you tested both of course.

Reply:
It shouldn't create a blank alias when `undefined` but I'm updating it to match the logic around jmesPath anyway, for consistency's sake.